### PR TITLE
Added verify target to Makefile

### PR DESCRIPTION
**Pull request by zvonkok**
Adding verify target to Makefile to automatically format go code with `gofmt`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,5 +48,19 @@ e2e-test:
 	dep ensure -v
 	go test -v ./test/e2e/ -args -nfd.repo=$(IMAGE_REPO) -nfd.tag=$(IMAGE_TAG_NAME) -kubeconfig=$(KUBECONFIG)
 
+verify:	verify-gofmt
```
Comment (Ethyling) [inline Makefile]: Some details, but I think you should replace this tab by a space

File: Makefile [hunk 2]
```diff
@@ -48,5 +50,19 @@ e2e-test:
...
```
Comment (zvonkok) [inline Makefile]:
```suggestion
verify: verify-gofmt
```

File: Makefile (hunk 1 above)
Comment (zvonkok) [reply Makefile]: Done.

File: Makefile [hunk 3]
```diff
@@ -15,6 +15,8 @@ IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
 K8S_NAMESPACE := kube-system
 KUBECONFIG :=
 
+GOFMT_CHECK=$(shell find . -not \( \( -wholename './.*' -o -wholename '*/vendor/*' \) -prune \) -name '*.go' | sort -u | xargs gofmt -s -l)
```
Comment (marquiz) [inline Makefile]: You're already doing the gofmt check here. Could that be moved to the actual make target? I.e. have something like `GO_FILES` here and then running `gofmt -s -l $(GO_FILES)` in the make target

File: Makefile [hunk 4]
```diff
@@ -48,5 +50,19 @@ e2e-test:
 	dep ensure -v
 	go test -v ./test/e2e/ -args -nfd.repo=$(IMAGE_REPO) -nfd.tag=$(IMAGE_TAG_NAME) -kubeconfig=$(KUBECONFIG)
 
+verify:	verify-gofmt
+
+verify-gofmt:
```
Comment (marquiz) [inline Makefile]: .PHONY target as well(?)

File: Makefile (hunk 2 above)
Comment (marquiz) [inline Makefile]: I think this should be a .PHONY target

**Pull request closed without merging**